worlds/hades/Options.py

```diff
@@ -0,0 +1,642 @@
+    Helpers give a boost to your max Health or boost the chance of obtaining rare Boons.
+    """
+    display_name = "Filler Helper Percentage"
+    range_start = 0
+    range_end = 100
+    default = 0
+    internal_name = "FillerHelperPercentage"
+
+class MaxHealthHelperPercentage(Range):
+    """
+    Choose the percentage of helper items that will boost your max health.
+    """
+    display_name = "Max Health Helper Percentage"
+    range_start = 0
+    range_end = 100
+    default = 35
+    internal_name = "MaxHealthHelperPercentage"
+
+class InitialMoneyHelperPercentage(Range):
+    """
+    Choose the percentage of helper items that will boost your initial money by 25 each run.
+    This gets capped by the percentage being left from the MaxHealthHelpers. 
+    What percentage remains from this and the MaxHealthHelpers will give you items that boost the 
+    rarity of the boons obtained in runs.
+    """
+    display_name = "Max Health Helper Percentage"
+    range_start = 0
+    range_end = 100
+    default = 35
+    internal_name = "InitialMoneyHelperPercentage"
+
+# -----------------------Settings for Trap -------------------------
+
+class FillerTrapPercentage(Range):
+    """
+    Choose the percentage of filler items in the pool that will be traps instead. 
+    Traps diminish your money or health during a run.
+    """
+    display_name = "Filler Trap Percentage"
+    range_start = 0
+    range_end = 100
+    default = 5
+    internal_name = "FillerTrapPercentage"
+
+# -----------------------Settings for QoL -------------------------
+
+class ReverseOrderExtremeMeasure(Toggle):
+    """
+    When true the order in which extreme meassures applied is reverse 
+    so level 1 is applied to Hades, instead to Meg/The Furies). 
+    For a more balanced experience.
+    """
+    display_name = "Reverse Order Extreme Measure"
+    option_true = 1
+    option_false = 0
+    default = 1
+    internal_name = "ReverseOrderExtremeMeasure"
+
+class IgnoreGreeceDeaths(Toggle):
+    """
+    If deaths on Greece are ignored for deathlink. Leave off for the memes.
+    """
+    display_name = "Ignore Greece Deaths"
+    option_true = 1
+    option_false = 0
+    default = 1
+    internal_name = "IgnoreGreeceDeaths"
+
+class StoreGiveHints(Toggle):
+    """
+    If seeing an item on the House Contractor's shop/Fated List of Prophecies 
+    should give a hint for it on the multiworld.
+    """
+    display_name = "Store Give Hints"
+    option_true = 1
+    option_false = 0
+    default = 1
+    internal_name = "StoreGiveHints"
+
+class AutomaticRoomsFinishOnHadesDefeat(Toggle):
+    """
+    If defeating Hades should give all room clears on Room based location mode 
+    or all rooms clears with the equipped weapon on Room weapon based location mode. 
+    """
+    display_name = "Automatic Room Finish On Hades Defeat"
+    option_true = 1
+    option_false = 0
+    default = 0
+    internal_name = "AutomaticRoomFinishOnHadesDefeat"
```

Comment:
`internal_name` doesn't seem to be doing anything.

Looks like it may have been copied from DLC Quest, Stardew, or TUNIC. TUNIC also doesn't use it. DLC Quest and Stardew use it in lieu of strings as entries in lists or dicts. I guess hoping to avoid problems where an option is mistyped as a string and the IDE can't tell you that something is wrong. But they're only able to do that because their `internal_name` attribute duplicates the attribute names in their respective options dataclasses, which Hades does not do.